apps/frontend/src/libs/components/button/styles.module.css

```diff
@@ -1,5 +1,6 @@
 .button {
 	padding: 10px 16px;
+	font-family: Inter, sans-serif;
```

Comment:
Failed to decode the downloaded font (http://localhost:3000/assets/fonts/inter/inter-medium.tt)

P.S. The regular and semi-bold are working fine.

Reply:
It was a typo mistake -> src: url("/assets/fonts/inter/inter-medium.tt") format("truetype");
Should be -> src: url("/assets/fonts/inter/inter-medium.ttf") format("truetype");

Can I fix this in this PR?

Reply:
yes, please